src/cloud-api-adaptor/ibmcloud/README.md

```diff
@@ -103,7 +108,7 @@ If you don't have a Kubernetes cluster for testing, you can follow the open-sour
 
 - Advertise the extended resource `kata.peerpods.io/vm.` by running the following commands:
   ```
-  pushd webhook/hack/extended-resources
+  pushd src/webhook/hack/extended-resources
```

Comment:
This is no longer needed as cloud-api-adaptor itself will publish the extended resources

Reply:
Thanks, good to know. I'm not sure if I should remove that in this PR or leave improvement to the README for another PR?

@stevenhorsman WDYT?

Reply:
Feel free to either create a separate commit in this PR that covers it, or you could check an issue on the repo and we have handle it later.

Reply:
Looks good. Thanks!

Reply:
done in a separate commit.